tests/test_lib/test_os.xsh

```diff
@@ -24,6 +24,8 @@ def test_indir():
 
 
 def test_rmtree():
+    if ON_WINDOWS:
+        pytest.skip("On Windows")
```

Comment:
I think we have `@skip_if_on_windows` decorator for this.

Reply:
This is xsh file. So I am not sure that works.

Reply:
This paradigme was used elsewhere in `test_xxx.xsh` files.

https://github.com/xonsh/xonsh/pull/3673/files#diff-1e0c5474373de0df12e386bdcc371662R12-R13